diktat-gradle-plugin/src/main/kotlin/com/saveourtool/diktat/plugin/gradle/tasks/SarifReportMergeTask.kt

```diff
@@ -33,19 +32,25 @@ abstract class SarifReportMergeTask : DefaultTask(), VerificationTask {
      */
     @get:InputFiles
     @get:PathSensitive(PathSensitivity.RELATIVE)
+    @get:Optional
     abstract val input: ConfigurableFileCollection
 
     /**
      * Destination for the merged report
      */
     @get:OutputFile
+    @get:Optional
     abstract val output: RegularFileProperty
 
     /**
      * @throws TaskExecutionException if failed to deserialize SARIF
      */
     @TaskAction
     fun mergeReports() {
+        if (!output.isPresent) {
+            logger.debug("Skipping merging SARIF reports because output is not set")
```

Comment:
?
```suggestion
            logger.warn("Skipping merging SARIF reports because output is not set")
```

Reply:
it's not a warn, we create `sarifReportMergeTask` each time, so input and output can be missed when sarif is not configured.
I didn't manage to create the task `sarifReportMergeTask` only when `sarif` (or `gitHubActions`) reporter is enabled.
But will try to do again :)

Reply:
okay, nevermind

Reply:
failed to resolve it quickly. will close an issue for it